### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1][WIP] List items: UI Type not read

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11256]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/core/list
UI element type for all the list item available is not communicated to the user by screen reader

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader can check by implementing similar case. Just use list item and try to read its by screen reader. All items of list are not readable by screen reader

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

**Comment by droshev** [comment]
@JyotiKumari1507 Can you provide more information about use-case, scenario, steps, whether you tried using the API for achieving the target?

**Comment by JyotiKumari1507** [comment]
@droshev  Screen reader isn't able to read all element of list while testing by Accessibility team. Its read only few 7-8 row of list which is visible in screen other item which comes after scrolling is missed out. We can have more discussion in next call

**Comment by JyotiKumari1507** [comment]
@droshev Attaching screen reader screenshot for more details:
<img width="903" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/157268666/be50b504-9ac0-49ef-8b14-f210823dabda">

**Comment by droshev** [comment]
Most of the fields are not filled.

**Comment by JyotiKumari1507** [comment]
@droshev please let us know what fields are pending to fill from our side

**Comment by droshev** [comment]
@JyotiKumari1507 did you set all aria attributes needed? If so, can you create a small stackblitz example demonstrating that you cannot set this on the application side?

**Comment by JyotiKumari1507** [comment]
@divyamalhotra488 please check on this from our side and share stackblitz example

**Comment by droshev** [comment]
@JyotiKumari1507 any update?

**Comment by divyamalhotra488** [comment]
Hi @droshev, Please find the stackblitz example here: https://stackblitz.com/edit/eclhkf?file=package.json

**Comment by mikerodonnell89** [comment]
Closing as nowhere in the issue or comments does not explain what needs to be done
